### ISSUE TITLE
Lift huge file limit or add specific error message

### ISSUE THREAD

**Issue by lachs0r** [Aegisub/Aegisub#110]
When attempting to open ASS subtitles above a certain size (100MB?), Aegisub will fail with a useless error message (“Cannot convert from utf-8 to binary”). It has no problem saving such files however.

Whether or not such files are sane, Aegisub should handle this case a little better and at least give meaningful errors/warnings (also when saving such files). I’m in favor of just removing the limit altogether.

**Comment by wangqr** [comment]
Yes the limit is 100MB, and is defined [here](https://github.com/Aegisub/Aegisub/blob/master/libaegisub/common/charset.cpp#L34).

**Comment by Carmaiquel** [comment]
I have a 3MB file and throws the same message, do you know how can I solve it?

**Comment by wangqr** [comment]
@Carmaiquel This can happen if your file is not in UTF-8-BOM encoding. Try use the `File -> Open Subtitles with Charset` and manually specify the encoding the file is using.
